scraper.go

```diff
@@ -31,11 +31,17 @@ func (s *scraper) Scrape(path string) ([]byte, error) {
 		},
 	}
 
-	response, err := client.Get(fmt.Sprintf("%v/%s", s.uri, path))
+	//response, err := client.Get(fmt.Sprintf("%v/%s", s.uri, path))
```

Comment:
you can remove the comment if we don't use it anymore

Reply:
I just remove the comment.